Android/app/src/main/java/app/intra/AutoStarter.java

```diff
@@ -0,0 +1,38 @@
+package app.intra;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.net.VpnService;
+import android.util.Log;
+import com.google.firebase.crash.FirebaseCrash;
+
+/**
+ * Broadcast receiver that runs on boot.
+ */
+
```

Comment:
nit: this also runs when the app restarted due to an update; extra newline here.

Reply:
Fixed.